spec/models/cycle_learning_unit_spec.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: cycle_learning_units
+#
+#  id               :bigint           not null, primary key
+#  cycle_id         :integer          not null
+#  learning_unit_id :integer          not null
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe CycleLearningUnit, type: :model do
+  context 'when creating an object with the factory' do
+    it do
```

Comment:
It would be better if you write a short description of the expected output in all test cases, for example `it "creates a valid record" do`.
Consider this comment for all spec files you edited.